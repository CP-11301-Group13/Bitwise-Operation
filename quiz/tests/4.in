100 2
65525 65324 65317 65534 65403 65528 65469 65497 65293 65456 65463 65378 65342 65426 65297 65392 65379 65508 65371 65394 65285 65303 65452 65392 65303 65425 65319 65531 65371 65319 65406 65289 65510 65292 65337 65289 65326 65433 65360 65382 65409 65491 65512 65293 65417 65313 65366 65433 65383 65368 65399 65321 65504 65504 65300 65378 65468 65415 65418 65461 65386 65353 65372 65415 65438 65370 65324 65495 65416 65490 65324 65368 65348 65380 65338 65400 65298 65370 65451 65294 65370 65349 65503 65387 65349 65471 65468 65421 65320 65388 65336 65396 65414 65350 65316 65437 65310 65506 65503 65296