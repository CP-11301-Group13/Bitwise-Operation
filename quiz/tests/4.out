65525 65411 65413 65531 65502 65522 65511 65398 65287 65504 65517 65432 65483 65384 65348 65488 65436 65457 65374 65496 65285 65357 65443 65488 65357 65380 65421 65534 65374 65421 65499 65286 65465 65283 65478 65286 65419 65382 65360 65433 65316 65404 65458 65287 65318 65412 65369 65382 65437 65362 65501 65414 65456 65456 65345 65432 65507 65325 65322 65509 65434 65302 65363 65325 65387 65370 65411 65405 65314 65400 65411 65362 65297 65425 65482 65490 65352 65370 65454 65291 65370 65301 65407 65438 65301 65519 65507 65319 65410 65427 65474 65489 65321 65305 65409 65383 65355 65464 65407 65344